### PR TITLE
Dev #767 allow subclipse wizards

### PR DESCRIPTION
**Pull request by Juancete**
Well.....Subclipse has a bug. When you try to share a new project, the wizard page is empty. But if you resize it, show the content. Then @javierfernandes says "have you tried with subversive?". So i made the change and works fine. Besides all the menues are not filtered.

### CODE REVIEW

File: org.uqbar.project.wollok.product.ui/src/org/uqbar/project/wollok/product/ui/startup/RemoveWizardsStartup.xtend [hunk 1]
```diff
@@ -42,7 +42,7 @@ class RemoveWizardsStartup implements WollokUIStartup {
 		val categories = registry.rootCategory.categories
 		for (wizard : categories.allWizards) {
 	        if (!wizard.id.includeWizards) {
-//	        	println("Removing wizard " + wizard.id)
+	        	//println("Removing wizard " + wizard.id)
```
Comment (tesonep) [inline org.uqbar.project.wollok.product.ui/src/org/uqbar/project/wollok/product/ui/startup/RemoveWizardsStartup.xtend]: It is a minor change, but remove the commented lines, they are in the git repo if we need it back (I think this println won't be needed).

**Comment by tesonep** [comment]
I think the change is ok, I have tested and it is working.  About the menus...

@fdodino what menus does you want or not want?
@Juancete do you think it can be configured easily?

**Comment by javierfernandes** [comment]
Can we merge this ? @Juancete 
Or are you doing some other changes ?

**Comment by fdodino** [comment]
I promise to test it. Today sourceforge was back to life, so I finished testing.
The only drawback I've seen is subversive authentication window for a repository, you should resize it otherwise user/password is hidden.
Ready to merge now (resolving conflicts)

**Comment by javierfernandes** [comment]
Damn ! wasn't that the same issue as with subclipse ? :(

**Comment by fdodino** [comment]
Not the same, you see the window with widgets, but not all of them appear. Well, important ones... user and password. But subclipse was worse than subversive. Juan is also explaining all in a well documented tutorial.

**Comment by javierfernandes** [comment]
http://s2.quickmeme.com/img/45/45439cd6d55d18f83d442376405d61d076fd70bcef2f56b231e8ec853324fdab.jpg

**Comment by fdodino** [comment]
jajajajajaja

**Comment by fdodino** [comment]
By the way... https://docs.google.com/document/d/1a0s0LA1foQOBsNalmBTvsRzq-UjGk8_tIO-iwr3JQuo/edit?usp=sharing
It's in progress

**Comment by Juancete** [comment]
Yes, it's ok to merge! Dodain give thumbs up!

**Comment by fdodino** [comment]
:+1: :+1:

**Pull request merged**